tests/Strategies/CallbackMaskingStrategyTest.php

```
@@ -235,7 +235,12 @@ public function testCallbackReceivesOriginalValue(): void
 
     public function testCallbackCanReturnNonString(): void
     {
-        $callback = fn(mixed $value): array => [TestConstants::DATA_MASKED => true];
+        $callback = /**
+         * @return true[]
+         *
+         * @psalm-return array{masked: true}
+         */
+        fn(mixed $value): array => [TestConstants::DATA_MASKED => true];
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$value'. (undefined)

(UnusedFormalParameter)